docs/kyma-environment-broker/03-08-kyma-versions.md

```diff
@@ -52,3 +53,5 @@ You can also specify a Kyma version using the **kymaVersion** provisioning param
 
 The **kymaVersion** provisioning parameter overrides the default settings.
 To enable this feature, set the **APP_ENABLE_ON_DEMAND_VERSION** environment variable to `true`.
+
+**Note:** If a Preview Plan is set, `kymaPreviewVersion` can be set to provision a preview version of Kyma. Alternatively `APP_KYMA_PREVIEW_VERSION` can be set as an environment variable.
```

Comment:
maybe something like this: "The default Kyma version for preview plan is a separate setting (APP_KYMA_PREVIEW_VERSION environment variable)"